scripts/zkai_seq32_derived_d128_mlp_surface_gate.py

```
@@ -0,0 +1,1027 @@
+
+
+def main() -> int:
+    args = parse_args()
+    payloads = build_component_payloads()
+    if args.write_inputs:
+        write_inputs(payloads)
+    if args.write_json or args.write_tsv:
+        write_summary(args.write_json, args.write_tsv)
+    mutation_result = run_mutations()
+    if not mutation_result["all_mutations_rejected"]:
+        raise Seq32DerivedD128MlpSurfaceError("mutation inventory accepted an invalid payload")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)